02-esc/challenges.js

```diff
@@ -1,143 +1,294 @@
-            setRating_star_5.classList.add('rating-star');
-            setRating.appendChild(setRating_star_5);
-            
-            if (rating < 1) {
-                setRating_star_1.classList.add('rating-star');
-            } else if (rating == 1 || rating < 2) {
-                setRating_star_1.classList.add('active');
-            } else if (rating == 2  || rating < 3) {
-                setRating_star_1.classList.add('active');
-                setRating_star_2.classList.add('active');    
-            } else if (rating == 3  || rating < 4) {
-                setRating_star_1.classList.add('active');
-                setRating_star_2.classList.add('active');
-                setRating_star_3.classList.add('active');
-            } else if (rating == 4  || rating < 5) {
-                setRating_star_1.classList.add('active');
-                setRating_star_2.classList.add('active');
-                setRating_star_3.classList.add('active');
-                setRating_star_4.classList.add('active');
-            } else {
-                setRating_star_1.classList.add('active');
-                setRating_star_2.classList.add('active');
-                setRating_star_3.classList.add('active');
-                setRating_star_4.classList.add('active');
-                setRating_star_5.classList.add('active'); 
-            }
-            
-            challenge_item.appendChild(setRating);
-            
-            let setTitle = document.createElement("h3");
-            setTitle.classList.add('challenge-title');
-            setTitle.textContent = title;
-            challenge_item.appendChild(setTitle);
-
-            let setParticipants = document.createElement("small");
-            setParticipants.classList.add('challenge-meta');
-            setParticipants.innerHTML = `${minParticipants} - ${maxParticipants} participants`;
-            challenge_item.appendChild(setParticipants);
-
-            let setDescription = document.createElement("p");
-            setDescription.classList.add('challenge-description');
-            setDescription.textContent = description;
-            challenge_item.appendChild(setDescription);
-
-            let btnBook = document.createElement("button");
-            btnBook.classList.add('btnBook');
-
-            btnBook.setAttribute("data-id", id); 
-                
-                if (type == 'onsite') {
-                    btnBook.textContent = 'Take challenge online';
-                } else {
-                    btnBook.textContent = 'Book this room';
-                }
-            challenge_item.appendChild(btnBook);
-
-            btnBook.addEventListener("click", function(){
-                console.log(id);
-                // referens till bokningsfunktion
-            });
-
-    
-            challenge_list.appendChild(challenge_item);
-            
-            console.log(title);
-      
-    };
-
-
-
-}  
\ No newline at end of file
+
```

Comment:
The fact that almost this entire file has changed makes it difficult to review. The reason it has changed seems to be formatting inconsistencies. There's a lesson here about making sure that all members of a team use consistent formatting. 😊

Reply:
For sure. Lesson learned about automatic formatters when doing group work. 🤦🏻‍♀️